operations/tasks/update-node-selector-labels.yml

```diff
@@ -0,0 +1,22 @@
+---
+# delete all predefined node selector labels which are not needed anymore
+- name: Remove managed node labels
+  command: >
+    oc label node {{ inventory_hostname }} {{ item }}-
+  with_items: "{{ osbs_managed_node_labels | difference(osbs_node_labels) }}"
+  register: delete_label
+  changed_when: ('not found' not in delete_label.stderr)
+  environment: "{{ osbs_environment }}"
+  tags:
+  - oc
+
+# set specified node selector labels
+- name: Apply node labels
+  command: >
+    oc label node {{ inventory_hostname }} {{ item }}
+  with_items: "{{ osbs_node_labels | intersect(osbs_managed_node_labels) }}"
```

Comment:
With only the items that are present in both `osbs_node_labels` and `osbs_managed_node_labels`? Sure that doesn't want to be `union(...)`?